Dockerfile

```diff
@@ -62,7 +62,8 @@ ARG TARGETPLATFORM
 # See for details: https://github.com/hadolint/hadolint/wiki/DL4006
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 # Create a user to run Photoview server
-RUN useradd -r -U -m photoview \
+RUN groupadd -g 999 photoview \
+  && useradd -r -u 999 -g photoview -m photoview \
```

Comment:
What implications does this have on existing setups. Could this break existing setups such that Photoview no longer has access to the files?

Reply:
Right now the image is created with the user and group, having UID=999 and GID=999, but as the command gets the 1st free ID, there is no guarantee that in the future after base image update these IDs won't change. That is why even while users see the IDs inside the container, they can't rely on them when managing FS permissions on the host. 

With this change, the IDs would be frozen/hardcoded, but still the same as currently used. This means, that those, who already configured their permissions on the host with the current IDs, will not see any difference, while all others will be able to do the same and rely on the IDs. If in the future at least 1 of the IDs is locked and not available at the build time, the image build will fail and we'll get the explicit error about this issue instead of automatically using some other ID and breaking users' setup.